modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java

```diff
@@ -2614,7 +2614,7 @@ private void _startWorkflowInstance(
 
 	private void _updateKBArticleAsset(
```

Comment:
inline this method on another commit

Reply:
It's used twice

Reply:
one directly from the `updateKBArticleAsset` and the other can call directly to the that.

no?

Reply:
You are right! I missed that.